packages/jest-matcher-utils/src/deepCyclicCopyReplaceable.ts

```diff
@@ -42,6 +44,9 @@ export default function deepCyclicCopyReplaceable<T>(
     return deepCyclicCopyMap(value, cycles);
   } else if (isBuiltInObject(value)) {
     return value;
+  } else if (plugins.DOMElement.test(value)) {
+    //@ts-expect-error skip casting to Node
+    return value.cloneNode(true);
```

Comment:
I need a suggestion.
I don't think use `@ts-expect-error` to ignore `Property 'cloneNode' does not exist on type 'T'` is a good idea.